packages/venia-ui/lib/components/FilterSidebar/filterSidebar.shimmer.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { shape, string } from 'prop-types';
+import { useStyle } from '../../classify';
+
+import Shimmer from '../Shimmer';
+import defaultClasses from './filterSidebar.css';
+
+const FilterSidebar = props => {
+    const classes = useStyle(defaultClasses, props.classes);
+
+    return (
+        <aside className={classes.root} aria-live="polite" aria-busy="true">
+            <Shimmer width="95%" height="70vh" style={{ marginBottom: 25 }} />
```

Comment:
Why `95%` I am not sure if use sizing like this

Reply:
@larsroettig It was to give a gap between the items and the sidebar, since padding will still have the background applied over that area. I can wrap the shimmer in an element with padding though if the `95%` is concerning for you!